package.json

```diff
@@ -79,7 +79,7 @@
     "@leafygreen-ui/menu": "18.0.0",
     "@leafygreen-ui/modal": "6.1.2",
     "@leafygreen-ui/palette": "3.4.4",
-    "@leafygreen-ui/popover": "7.2.2",
+    "@leafygreen-ui/popover": "11.0.0",
     "@leafygreen-ui/radio-box-group": "12.0.0",
     "@leafygreen-ui/radio-group": "10.0.0",
```

Comment:
This one random tooltip is in `darkMode`, would you mind taking the prop off for it 😅

<img width="345" alt="Screen Shot 2022-11-18 at 5 01 49 PM" src="https://user-images.githubusercontent.com/47064971/202810622-859493da-c0ca-4f69-87df-3efe712c425b.png">